spec/requests/actions_controller_spec.rb

```diff
@@ -84,7 +89,7 @@
     }
     enums_action_definition = {
       name: 'enums_action',
-      fields: [foo, enum],
+      fields: [foo, enum, multiple_enum],
```

Comment:
Why is `multiple_enum` in `enums_action_definition`?
Can we keep it separated?

`enums_action_definition` with `fields: [foo, enum],`
and
`multiple_enums_action_definition` with `fields: [foo, multiple_enum],`

Would be easier to read and review :)

Reply:
True ! I'm changing this